test/apps/iodemo/io_demo.cc

```diff
@@ -742,20 +795,56 @@ class DemoServer : public P2pDemoCommon {
         memory_pin_stats_t pin_stats;
         memory_pin_stats(&pin_stats);
 
-        LOG << "read:" << _curr_state.read_count - _prev_state.read_count
+        conn_stat_map_t::iterator it_min = _conn_stat_map.begin(),
+                                  it_max = _conn_stat_map.begin();
+        for (conn_stat_map_t::iterator it = _conn_stat_map.begin();
+             it != _conn_stat_map.end(); ++it) {
+
+            long i_val = it->second.get_total_count();
+
+            if (i_val <= it_min->second.get_total_count()) {
+                it_min = it;
+            } else if (i_val >= it_max->second.get_total_count()) {
+                it_max = it;
+            } else {
+                it->second.reset();
+            }
+        }
+
+        UcxLog log(LOG_PREFIX);
+        log << "read:" << _curr_state.read_count - _prev_state.read_count
             << " ops, "
             << "write:" << _curr_state.write_count - _prev_state.write_count
             << " ops, "
-            << "active connections:" << _curr_state.active_conns
-            << ", buffers:" << _data_buffers_pool.allocated()
+            << "active connections:" << _curr_state.active_conns;
+
+        if (_curr_state.active_conns > 0) {
+            assert(!_conn_stat_map.empty());
+
+            log << ", min:" << it_min->second.get_total_count() << "/"
+                << it_min->second.get_total_bytes() / double(UCS_MBYTE)
+                << "MB, (" << get_connection(it_min->first).get_peer_name()
```

Comment:
shall we use get_peer_name in DemoClient as well?

Reply:
on client side we use `server_name(index)` which goes from `options_t test_opts;`, I think that's OK